<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
  <class>GUI_AddAuthor</class>
  <widget class="QDialog" name="GUI_AddAuthor">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>125</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>125</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>300</width>
        <height>125</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Добавление автора</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
            <widget class="QLineEdit" name="fatherNameText" />
          </item>
          <item row="1" column="2">
            <widget class="QLineEdit" name="nameText" />
          </item>
          <item row="0" column="2">
            <widget class="QLineEdit" name="surnameText" />
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="nameLabel">
              <property name="text">
                <string>Имя:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="surnameLabel">
              <property name="text">
                <string>Фамилия:</string>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="fatherNameLabel">
              <property name="text">
                <string>Отчество:</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLabel" name="nameMUST">
              <property name="styleSheet">
                <string notr="true">color : red;</string>
              </property>
              <property name="text">
                <string notr="true">*</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLabel" name="surnameMUST">
              <property name="styleSheet">
                <string notr="true">color : red;</string>
              </property>
              <property name="text">
                <string notr="true">*</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>surnameText</tabstop>
    <tabstop>nameText</tabstop>
    <tabstop>fatherNameText</tabstop>
  </tabstops>
  <resources />
  <connections />
</ui>